packages/components/src/internal/app/utils.ts

```diff
@@ -273,6 +274,22 @@ export function isProductProjectDataTypeSelectionEnabled(moduleContext?: ModuleC
     return resolveModuleContext(moduleContext)?.query?.[EXPERIMENTAL_PRODUCT_PROJECT_DATA_SELECTION] === true;
 }
 
+export function getProjectDataExclusion(moduleContext?: ModuleContext): { [key: string]: number[] } {
+    return resolveModuleContext(moduleContext)?.samplemanagement?.[PROJECT_DATA_TYPE_EXCLUSIONS];
+}
+
+export function getProjectSampleTypeExclusion(moduleContext?: ModuleContext): number[] {
+    return getProjectDataExclusion(moduleContext)?.['SampleType'];
```

Comment:
case shouldn't be a problem as those are the result of a controller api action. 
Good thought on empty array. But because there is also use of the generic getProjectDataExclusion when different exclusions are needed, I feel it's slightly safer to not assume null in all cases.